src/plots/Vector/VectorAttributes.code

```diff
@@ -215,6 +144,110 @@ VectorAttributes::ProcessOldVersions(DataNode *parentNode,
         if (searchNode->GetNode("lineStyle") != 0)
             searchNode->RemoveNode("lineStyle");
     }
+
+    if (VersionLessThan(configVersion, "3.2.0"))
+    {
+        DataNode *k = 0;
+        if((k = searchNode->GetNode("colorByMag")) != 0)
+	{
```

Comment:
Whitespace nit. You've introduced tabs With the opening and closing braces for the internal if.